CHANGELOG.md

```diff
@@ -21,19 +23,27 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
     - This fixes odd issues which can occur when mounting over an existing
       mount point (we mount `.` as `/opt/openoni`, and previously were mounting
       `./docker/data` as `/opt/openoni/data`, effectively shadowing the actual data directory)
-  - All generated ingest artifacts now live in a named volume
+  - All generated ingest artifacts now live in a named volume, `onidata`, which
+    is mounted into multiple containers as `/var/local/onidata`
 
 ### Migration
 
 - If you use any of our plugins, make sure you look over their repositories and
-  get a version that is built for Python 3
-- Custom themes and plugins you've built will need to be fixed for Python 3 -
-  exact fixes are out of scope for ONI, but you can learn a lot from the
-  [Python porting documentation](https://docs.python.org/3/howto/pyporting.html)
-  as well as the [2to3](https://docs.python.org/2/library/2to3.html) tool's
-  documentation.
+  get a version that is built for Django 2.2.  Django 2.2 only supports Python
```

Comment:
Should this be modified any to tell where to look in the plugin README's etc, now that you've updated those?

Reply:
Agreed.  Gimme a bit and I'll fix that.

Reply:
Fixed!